aten/src/ATen/cpu/vec256/vec256_int.h

```diff
@@ -95,25 +95,25 @@ struct Vec256<int64_t> : public Vec256i {
     return _mm256_cmpeq_epi64(values, other.values);
   }
   Vec256<int64_t> operator!=(const Vec256<int64_t>& other) const {
-    auto zero = _mm256_set1_epi64x(0);
+    static const auto one = _mm256_set1_epi64x(-1);
     auto eq = _mm256_cmpeq_epi64(values, other.values);
-    return _mm256_xor_si256(zero, eq);  // invert
+    return _mm256_xor_si256(one, eq);  // invert
```

Comment:
I think you're right about this. Maybe it would be nicer if we refactor the inverting logic into its own `static inline` function?

Also, it's a little nicer if we call it `ones` instead of `one` (because the bits are all ones; they don't represent the number one) if I'm understanding this correctly